README.md

```diff
@@ -46,6 +46,10 @@ If you want to use a package manager:
 - [Snap](https://snapcraft.io/) users can use `sudo snap install helm
   --classic`.
 
+If you want to install helm from script:
+
+- [Script](https://github.com/helm/helm/blob/master/docs/install.md#from-script) instructions to install from script on a new instance without Helm. For example, you want to install on a container, cluster, etc. Note: GCP console comes with `Helm` installed by default.
```

Comment:
To make it more generic, I would drop the specific mention to GCP and say something like

> Note: Many development environments available in public clouds may already have `helm` installed. Make sure to check before installing

Reply:
Marking as unresolved as this has not been addressed.